packages/sanity/src/desk/comments/src/hooks/use-mention-options/helpers.ts

```diff
@@ -0,0 +1,72 @@
+import {SanityDocument} from '@sanity/client'
+import {evaluate, parse} from 'groq-js'
+import {MentionOptionUser} from '../../types'
+import {EvaluationParams, Grant, DocumentValuePermission, PermissionCheckResult} from 'sanity'
+
+// TODO:
+// This code is copied from the `grantsStore.ts` and slightly modified to work with the `MentionOptionUser` type.
+// TODO: Refactor this to be a shared function between the `grantsStore.ts` and this file.
```

Comment:
+1 do we/should we have a matching ticket for this?

Reply:
Yes, I'll create a ticket for this!